devops/resources/user-retirement-archiver.sh

```diff
@@ -22,4 +22,5 @@ pip install -r requirements.txt
 # Call the script to read the retirement statuses from the LMS, send them to S3, and delete them from the LMS.
 python scripts/retirement_archive_and_cleanup.py \
     --config_file=$WORKSPACE/user-retirement-secure/${ENVIRONMENT_DEPLOYMENT}.yml \
-    --cool_off_days=$COOL_OFF_DAYS
+    --cool_off_days=$COOL_OFF_DAYS \
+    --batch_size=$BATCH_SIZE
```

Comment:
Don't you need to add the BATCH_SIZE parameter if you're adding it?

Reply:
Following along with this PR, I thought you meant add it to the script, but I confirmed it was added in https://github.com/edx/tubular/pull/530/files#diff-f17c99b35f9f5bad8724b41b2dd37caa3dafd9c9542bb6e6b9f3751de311755fR76, then I relaized you meant adding a parameter to the Jenkins job.  I agree that it might make sense to add a param to input the batch zer via manually triggered jobs in [here](https://github.com/edx/jenkins-job-dsl/pull/1426/files#diff-9441adf7e1a53035ec1d63faeade53e0bb89ef557ba0fda330e2e84d32a8b7deR77).